package.json

```diff
@@ -249,7 +249,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/component-library": "^3.10.1",
+    "@department-of-veterans-affairs/component-library": "^3.11.1",
```

Comment:
The screenshot was taken with this change as part of the build. I tried reverting it and running the same test, but there was a problem in the app entry file, not `web-components.entry.js`. I don't understand why that's happening.